### ISSUE TITLE
Failed tests by Secp256k1

### ISSUE THREAD

**Issue by monaka** [Internet2-WG/rust-aluvm#60]
My env is the same as #58.

Error log:

```
$ cargo test --no-fail-fast
   Compiling aluvm v0.6.0 (/home/monaka/rgb/rust-aluvm)
error[E0599]: no variant or associated item named `Secp256k1` found for enum `Instr` in the current scope
  --> examples/asm.rs:25:16
   |
25 |       let code = aluasm! {
   |  ________________^
26 | |         clr     r1024[5]                        ;
27 | |         put     5,a16[8]                        ;
28 | |         putif   0xaf67937b5498dc,r256[1]        ;
...  |
92 | |         ret                                     ;
93 | |     };
   | |_____^ variant or associated item not found in `Instr<_>`
   |
   = note: this error originates in the macro `instr` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0599`.
error: could not compile `aluvm` due to 4 previous errors
```

Is this issue related to #14 ?

**Comment by dr-orlovsky** [comment]
I think this should be related to the imports and not related to #14. It could also be a different features set. I will be able to investigate the issue in the following days

**Comment by dr-orlovsky** [comment]
Potentially closed by #61 - we can close this after checking

**Comment by monaka** [comment]
This issue is not resolved.

`cargo test --all-features` works.

But `cargo test` doesn't
